keyboards/kbdfans/kbd19x/kbd19x.c

```diff
@@ -16,3 +16,23 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
 #include "kbd19x.h"
+
+void keyboard_pre_init_kb(void) {
+    led_init_ports();
+    keyboard_pre_init_user();
+}
+
+void led_init_ports(void) {
+    setPinOutput(B0);
+    setPinOutput(B1);
+    setPinOutput(B2);
+}
+
+bool led_update_kb(led_t led_state) {
+    if (led_update_user(led_state)) {
+        writePin(B0, !led_state.num_lock);    // Left Indicator LED
+        writePin(B1, !led_state.caps_lock);   // Top Indicator LED
+        writePin(B2, !led_state.scroll_lock); // Right Indicator lED
+    }
+    return true;
+}
```

Comment:
```suggetion
```

If you add to config.h...
```c
#define LED_NUM_LOCK_PIN B0
#define LED_CAPS_LOCK_PIN B1
#define LED_SCROLL_LOCK_PIN B2
```

https://docs.qmk.fm/#/feature_led_indicators?id=configuration-options